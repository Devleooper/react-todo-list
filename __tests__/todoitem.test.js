import React from "react";
import renderer from "react-test-renderer";
import TodoItem from "../src/components/todoitem/todo.item";

it("renders correctly", () => {
  const item = renderer.create(<TodoItem />);
  expect(item.toJSON()).toMatchSnapshot();
});

it("renders correctly with text", () => {
  const item = renderer.create(<TodoItem text="this is a snapshot test" />);
  expect(item.toJSON()).toMatchSnapshot();
});
