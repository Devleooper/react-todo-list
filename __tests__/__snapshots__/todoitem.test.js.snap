// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="item-container"
>
  <h3 />
  <div
    className="side-btn"
  >
    <button
      className="delete-btn"
    >
      <span
        aria-label="delete"
        role="img"
      >
        ❌
      </span>
    </button>
  </div>
</div>
`;

exports[`renders correctly with text 1`] = `
<div
  className="item-container"
>
  <h3>
    this is a snapshot test
  </h3>
  <div
    className="side-btn"
  >
    <button
      className="delete-btn"
    >
      <span
        aria-label="delete"
        role="img"
      >
        ❌
      </span>
    </button>
  </div>
</div>
`;
